src/middlewares/userValidations.js

```diff
@@ -45,6 +45,34 @@ const emailExists = (req, res, next) => {
   });
 };
 
+/**
+ * @function
+ * @description Check if user email exist, password correct and verified
+ * @param {object} req - Resquest object
+ * @param {object} res - Response object
+ * @param {object} next
+ * @returns {object} JSON response
+ */
+const loginData = (req, res, next) => {
+  const { email, password } = req.body;
```

Comment:
This email should be trimmed and changed to lowercase
```suggestion
  const { password } = req.body;
  let { email } = req.body;
  email = email.trim().toLowerCase()
```

Reply:
Thanks. Changes effected